src/services/contracts/baseDAO/treasuryDAO/index.ts

```diff
@@ -1,32 +1,34 @@
-import {TezosToolkit} from "@taquito/taquito";
-import {Schema} from "@taquito/michelson-encoder";
-import {Expr, Parser} from "@taquito/michel-codec";
-import {BaseDAO, BaseDAOData, getContract} from "..";
-import {TreasuryProposeArgs} from "./types";
+import { TezosToolkit } from "@taquito/taquito";
+import { Schema } from "@taquito/michelson-encoder";
+import { Expr, Parser } from "@taquito/michel-codec";
+import { BaseDAO, BaseDAOData, getContract } from "..";
+import { TreasuryProposeArgs } from "./types";
 import proposeCode from "./michelson/propose";
-import {TreasuryExtraDTO} from "services/indexer/types";
-import {mapTransfersArgs} from "services/indexer/dao/mappers/proposal";
-import {BigNumber} from "bignumber.js";
-import {formatUnits} from "../../utils";
+import { TreasuryExtraDTO } from "services/indexer/types";
+import { mapTransfersArgs } from "services/indexer/dao/mappers/proposal";
+import { BigNumber } from "bignumber.js";
+import { formatUnits } from "../../utils";
 
 const parser = new Parser();
 
 interface TreasuryDAOData extends BaseDAOData {
   extra: TreasuryExtraDTO;
+  dao_type: {
+    id: number;
+    name: string;
+  };
```

Comment:
Same as above, this is specifically a Treasury interface, it should not contain a dao_type property